compiler-rt/cmake/Modules/CompilerRTUtils.cmake

```diff
@@ -369,7 +369,11 @@ macro(construct_compiler_rt_default_triple)
   endif()
 
   if ("${CMAKE_C_COMPILER_ID}" MATCHES "Clang")
-    execute_process(COMMAND ${CMAKE_C_COMPILER} --target=${COMPILER_RT_DEFAULT_TARGET_TRIPLE} -print-target-triple
+    set(cmd_prefix "")
+    if (CMAKE_C_SIMULATE_ID MATCHES "MSVC")
+      set(cmd_prefix "/clang:")
```

Comment:
This does what exactly?

I guess that it marks the following option as a normal clang style option `/clang: -something-like-this` instead of a `/option` which cl.exe would be expecting.

Reply:
Maybe call it "option prefix" instead, since to my eyes the "command" is the whole thing `clang ...`.

Reply:
> This does what exactly?
> 
> I guess that it marks the following option as a normal clang style option `/clang: -something-like-this` instead of a `/option` which cl.exe would be expecting.

Yes exactly as you said. It is used in case we want to use clang like argument with clang-cl.